src/reportSources/legacyFflogs/eventAdapter/translate.ts

```diff
@@ -123,16 +122,18 @@ export class TranslateAdapterStep extends AdapterStep {
 		}
 	}
 
-	private adaptSnapshotEvent(event: DamageEvent | HealEvent): Array<Events['snapshot' | 'actorUpdate']> {
+	private adaptFFLogsEffectEvent(event: DamageEvent | HealEvent): Array<Events['execute' | 'damage' | 'heal' | 'actorUpdate']> {
 		// Calc events should all have a packet ID for sequence purposes. Let sentry catch outliers.
 		const sequence = event.packetID
 		if (sequence == null) {
```

Comment:
possibly worth checking that it _is_ coming from a status? it's safe to check the offset const in here.
not sure if there's a special cases that'd make this dangerous or not (ground effects maybe?)

Reply:
Doton and Salted Earth are both being tracked as Status DOT ticks on FFLogs, so that sounds sane & safe to me.  Worst case I break prod again 🤣